data_steward/validation/main.py

```diff
@@ -730,15 +730,20 @@ def run_retraction_cron():
 
     # retract from bq
     dataset_ids = bq_utils.get_retraction_dataset_ids()
-    dataset_ids = dataset_ids.split() if dataset_ids else None
+    # If the env variable RETRACTION_DATASET_IDS does not exist, dataset_ids will be set to None
+    # This allows retraction to be performed on all datasets existing in the project
+    if dataset_ids is not None:
```

Comment:
nit:
```
if dataset_ids:
    dataset_ids = dataset_ids.split()
```

Reply:
Then this is a logic flaw.  If `dataset_ids` can be any type other than a string and get through this condition, the code will fail on the next line.  `split()` is a string attribute.  If you try to use it on a boolean, integer, or some other type you will get an `AttributeError`.  So you probably want to check:
```
if isinstance(dataset_ids, str):
    dataset_ids = dataset_ids.split()
```